app-i18n/ibus-libpinyin/ibus-libpinyin-1.11.1.ebuild

```diff
@@ -22,17 +21,20 @@ KEYWORDS="~amd64 ~x86"
 IUSE="boost lua opencc"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
-RDEPEND="${PYTHON_DEPS}
+BDEPEND="
+	virtual/pkgconfig"
+
+DEPEND="${PYTHON_DEPS}
 	app-i18n/ibus[python(+),${PYTHON_USEDEP}]
 	>=app-i18n/libpinyin-2.1.0:=
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	virtual/libintl
+	dev-db/sqlite:3
```

Comment:
If you moved this line 2 lines above, then dependencies would be sorted alphabetically :) .

Reply:
ok